README.md

```diff
@@ -1,36 +1,32 @@
-# Rapidez :package_name_without_prefix
-<!--delete-->
-This repository can be used as template for a new Rapidez package.
+# Rapidez vat-validation
 
-- Click on "Use this template" on the top of this Github repo page
-- Run `php ./configure.php`
+Implements VIES validation to use on VAT fields in Rapidez
 
-Credits to [`spatie/package-skeleton-laravel`](https://github.com/spatie/package-skeleton-laravel) for the inpiration for this template.
+## Installation
 
-Keep in mind that if you contribute to this template; it should work for official and unofficial packages!
-- `rapidez/something`
-- `someone/rapidez-something`
-<!--/delete-->
-:package_description
+```
+composer require rapidez/vat-validation
+```
 
-## Installation
+## Usage
 
+You can add the VAT check to an input by adding this to the input:
 ```
-composer require :vendor_slug/:package_slug
+v-on:change="window.app.$emit('vat-change', $event)"
```

Comment:
For sure, I'm waiting for this to be approved first and then I'll make a PR to the core, checkout theme, and account. Should still have this in the readme though, so that people can easily add it to their own fields :)

Reply:
https://github.com/rapidez/core/pull/607
https://github.com/rapidez/core/pull/606
https://github.com/rapidez/checkout-theme/pull/120
https://github.com/rapidez/account/pull/53